SADL/AM-CDM-build.sadl

```diff
@@ -1,135 +1,77 @@
-	described by amInSituData (note "Advanced AM in-process monitoring BIG data, is differentiated from processData.") with values of type AMInSituData,
- 	described by buildInterruptions (note "logs of build interruptions") with values of type BuildInterruption,
-	described by buildDuration (note "Recorded duration of build cycle.") with a single value of type Measurement,
-	described by buildLocation (note "Place of an AM process.") with a single value of type Address,
-	described by buildCooldownDuration (note "Time from build completion to cooling to 100 degrees C.") with a single value of type Measurement,
-	described by buildImages(note "Time from build completion to cooling to 100 degrees C.") with values of type Document,
-	described by buildNote (note "Time from build completion to cooling to 100 degrees C.") with a single value of type string.
+AMMaterialInput is a MaterialInputs,
+	described by feedstockMaterial (note "bulk raw material supplied to the additive manufacturing building process.") with a single value of type MaterialStock,
+	described by quantityOfFeedstockMaterialUsed (note "Mass of material used in solidified parts.") with a single value of type Quantity,
+	described by buildPlate with a single value of type BuildPlate.
 
-Part (note "Instances of part designs in the as-built state.") is a class,
-	described by partID (note "Identifier for the built part.") with a single value of type string,
-	described by partName (note "Conventional name for the built part.") with a single value of type string,
-	described by processHistory with a single value of type ProcessHistory,
-	described by partID (note "Identifier for the built part.") with a single value of type string,
-	described by partDesign (note "Design information for the built part.") with a single value of type PartDesign,
-	described by partProcessParameters (note "Part-specific, reusable process parameters associated with printing this part.") with values of type ProcessParameters,
-	described by partCoordinateOnBuildX (note "The location X of the box center of the part, or an accepted reference datum using the build coordinate system defined in ISO/ASTM 52921") with a single value of type Measurement,
-	described by partCoordinateOnBuildY (note "The location Y of the box center of the part, or an accepted reference datum using the build coordinate system defined in ISO/ASTM 52921") with a single value of type Measurement,
-	described by partCoordinateOnBuildZ (note "The location Z of the box center of the part, or an accepted reference datum using the build coordinate system defined in ISO/ASTM 52921") with a single value of type Measurement,
-	described by partOrientationOnBuild (note "The orientation defined in ISO 52921.") with a single value of type Orientation.
+// Does this replace the 'Build Product' concept we discussed at NIST

```

Comment:
I vote no, we should keep BuildProduct instead of AMMaterialOutput and maybe change AMMaterialInput to something else? BuildInput?  I think our reasoning for not using the word "material" is that the materials folks didn't like it?

Reply:
Implemented and agreed upon!